Jenkinsfile

```diff
@@ -38,24 +39,38 @@ def buildManifest = {String manifest, String bitbake_image ->
     }
 
     stage("Setup bitbake and do fetchall ${bitbake_image}") {
-        // Extract the BSP part of the manifest file name.
-        String bsp = manifest.split("-")[1].tokenize(".")[0]
         // Setup bitbake environment and trigger a 'fetchall'
-        sh "vagrant ssh -c \"TEMPLATECONF=${yoctoDir}/sources/meta-pelux-bsp-${bsp}/conf /vagrant/vagrant-cookbook/yocto/fetch-sources-for-recipes.sh ${yoctoDir} ${bitbake_image}\""
+        confdir = "conf"
+        if (qtauto) {
+            confdir += "-qt"
+        }
+        templateconf="${yoctoDir}/sources/meta-pelux/meta-${bsp}-extras/${confdir} "
+        sh "vagrant ssh -c \"TEMPLATECONF=${templateconf} /vagrant/vagrant-cookbook/yocto/fetch-sources-for-recipes.sh ${yoctoDir} ${bitbake_image}\""
     }
 
     stage("Bitbake ${bitbake_image}") {
         sh "vagrant ssh -c \"/vagrant/vagrant-cookbook/yocto/build-images.sh ${yoctoDir} ${bitbake_image}\""
     }
 
+    stage("Copy downloads and cache ${bitbake_image}") {
+        // Archive the downloads and sstate when the environment variable was set to true
+        // by the Jenkins job.
+        if (env.ARCHIVE_CACHE) {
+            sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/downloads/ /vagrant/archive/\""
+            sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/sstate-cache/ /vagrant/archive/\""
```

Comment:
This has been changed to use rsync instead in pyro/master, should be reflected here as well.